LION/models/learned_regularizer/ACR.py

```diff
@@ -61,7 +63,7 @@ class L2net(nn.Module):
     def __init__(self):
         super(L2net, self).__init__()
 
-        self.l2_penalty = nn.Parameter((-9.0) * torch.ones(1))
+        self.l2_penalty = nn.Parameter((-36.0) * torch.ones(1))
```

Comment:
lets make this value an input for now. We'll need @Zakobian to weigh in if there is a reasonable way to estimate this value.

Reply:
Another question with this, do we want these small modules to be LionModels? I'd suggest no, because as I view them, they're just extracting a bit of code form inside the ACR network. Doesn't seem grand enough

Reply:
@cshoebridge currently there is a serious issue with saving/loading that does not allow a LIONmodel inside a LIONmodel to be correctly saved. So for now, definetly no.